src/client/config/dev-cloud.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "useEndPoint": true,
```

Comment:
I think an 'overwrite' concept seems reasonable for now and we keep that closer to the dev workflow.